MUXControls.TestApp Interaction Tests fail to run if Windows SDK was installed in a custom location

**Describe the bug**
When trying to run [interaction tests](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/developer_guide.md#automated-testing) the operation fails because the referenced `Windows.winmd` file cannot be found. 

![image](https://user-images.githubusercontent.com/1398851/76245153-acce1180-623b-11ea-9d85-aa6a97f0f50e.png)

The error message VS gives me is the following: `C:\Program Files (x86)\Programming\IDEs\VS2019\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2106,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "Windows.winmd". Check to make sure the assembly exists on disk.`

**Why this happens**
I have installed the Windows 10 SDK in a custom location and not the standard `C:\Program Files (x86)\Windows Kits` location. Looking into the MUXControls.Test.Shared.projitems file I see the following line: `<WindowsMetadataPath>$(UniversalCRTSdkDir)\UnionMetadata\$(TargetPlatformVersion)</WindowsMetadataPath>` with `UniversalCRTSdkDir` defined as: `<UniversalCRTSdkDir Condition="'$(UniversalCRTSdkDir)' == ''">$(MSBuildProgramFiles32)\Windows Kits\10</UniversalCRTSdkDir>` (line directly above).

The [documentation](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-reserved-and-well-known-properties?view=vs-2019) for `MSBuildProgramFiles32` says:
![image](https://user-images.githubusercontent.com/1398851/76246217-8e691580-623d-11ea-95cc-60d67c0cf764.png)
 
So MUXControls.TestApp ends up looking in location `C:\Program Files (x86)\Windows Kits\10\UnionMetadata\$(TargetPlatformVersion)` for the Windows.winmd file, yet as I mentioned at the beginning, I installed the Windows 10 SDK in a non-standard location. 

In order to find the Windows metadata file and thus run the interaction tests, I am currently overwriting the definition of the UniversalCRTSdkDir entry in the `MUXControls.Test.Shared.projitems` file to point to the correct windows 10 SDK location (which in my case is D:\Windows Kits\10). Now that I changed that file I have to be careful not to accidentally include this change in every commit I will make to the WinUI repo (as it would break the setup for others). As such, it would be great if WinUI itself could update the file so that the Windows 10 metadata file will be found even if the SDK location differs from the standard location.

Note: There might also be a solution using VS. I've read that one can run %VSInstallDir%\Common7\Tools\VsDevCmd.bat file to set directories like `UniversalCRTSdkDir`. If this works (haven't tested it here) this could be an alternative to changing the MUXControls.Test.Shared.projitems file in case that should prove challenging. In this case, documentation should be added though in the [Testing](https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/developer_guide.md#testing) paragraph.

--- a/test/MUXControls.Test/MUXControls.Test.Shared.projitems
+++ b/test/MUXControls.Test/MUXControls.Test.Shared.projitems
@@ -26,6 +26,7 @@
     <LangVersion>$(CSLangVersion)</LangVersion>
   </PropertyGroup>
   <PropertyGroup Condition="$(BuildingWithBuildExe) != 'true'">
+    <UniversalCRTSdkDir Condition="'$(UniversalCRTSdkDir)' == ''">$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Kits\Installed Roots@KitsRoot10)</UniversalCRTSdkDir>
     <UniversalCRTSdkDir Condition="'$(UniversalCRTSdkDir)' == ''">$(MSBuildProgramFiles32)\Windows Kits\10</UniversalCRTSdkDir>
     <WindowsMetadataPath>$(UniversalCRTSdkDir)\UnionMetadata\$(TargetPlatformVersion)</WindowsMetadataPath>
     <AssemblyName>MUXControls.Test</AssemblyName>

--- a/test/MUXControls.Test/MUXControls.Test.Shared.projitems
+++ b/test/MUXControls.Test/MUXControls.Test.Shared.projitems
@@ -27,8 +27,8 @@
   </PropertyGroup>
   <PropertyGroup Condition="$(BuildingWithBuildExe) != 'true'">
     <UniversalCRTSdkDir Condition="'$(UniversalCRTSdkDir)' == ''">$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows Kits\Installed Roots@KitsRoot10)</UniversalCRTSdkDir>
-    <UniversalCRTSdkDir Condition="'$(UniversalCRTSdkDir)' == ''">$(MSBuildProgramFiles32)\Windows Kits\10</UniversalCRTSdkDir>
-    <WindowsMetadataPath>$(UniversalCRTSdkDir)\UnionMetadata\$(TargetPlatformVersion)</WindowsMetadataPath>
+    <UniversalCRTSdkDir Condition="'$(UniversalCRTSdkDir)' == ''">$(MSBuildProgramFiles32)\Windows Kits\10\</UniversalCRTSdkDir>
+    <WindowsMetadataPath>$(UniversalCRTSdkDir)UnionMetadata\$(TargetPlatformVersion)</WindowsMetadataPath>
     <AssemblyName>MUXControls.Test</AssemblyName>
     <!-- We are using .NetCore because we are targetting CoreCLR but with Windows UAP support. It maps to netcore50 in the project.json file. 
          If you don't need UAP support, for example you are just driving UI with Microsoft.Windows.Apps.Test, you should build with .NETPlatform,Version=5.4. This maps to dotnet54 in the project.json file.

